### PR TITLE
New package: Jpegoptim 1.4.4

### PR DESCRIPTION
**Pull request by rorgoroth**
Tested on w32 & w64.

### CODE REVIEW

File: mingw-w64-jpegoptim/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+pkgver=1.4.4
+pkgrel=1
+pkgdesc="Utility to optimize jpeg files"
+arch=('any')
+url="http://www.kokkonen.net/tjko/projects.html"
+license=('GPL')
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc")
+depends=("${MINGW_PACKAGE_PREFIX}-libjpeg-turbo")
+source=("https://www.kokkonen.net/tjko/src/${_realname}-${pkgver}.tar.gz")
+md5sums=('e8fe81629e2b27ba17c0d3ddd15bd782')
```
Comment (mati865) [inline mingw-w64-jpegoptim/PKGBUILD]: Use `sha256sums`

File: mingw-w64-jpegoptim/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.4.4
+pkgrel=1
+pkgdesc="Utility to optimize jpeg files"
+arch=('any')
+url="http://www.kokkonen.net/tjko/projects.html"
+license=('GPL')
+depends=("${MINGW_PACKAGE_PREFIX}-libjpeg-turbo")
+source=("https://www.kokkonen.net/tjko/src/${_realname}-${pkgver}.tar.gz")
```
Comment (Chocobo1) [inline mingw-w64-jpegoptim/PKGBUILD]: Validating PGP signature is encouraged, example:
https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-curl/PKGBUILD#L32
https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-curl/PKGBUILD#L39

File: mingw-w64-jpegoptim/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+sha256sums=('89a03ea2dfe9483dfb2e009a2851be0befb0deb7cb45c04550bcc91e1e88c1b2')
+
+build() {
+    cd "$srcdir/${_realname}-$pkgver"
+    ./configure --prefix=${MINGW_PREFIX}
+    make
+    strip jpegoptim.exe
```
Comment (mati865) [inline mingw-w64-jpegoptim/PKGBUILD]: Strip is default option so it should be stripped without adding anything.

Comment (rorgoroth) [inline mingw-w64-jpegoptim/PKGBUILD]: Thanks mati, when manually building `make strip` is borked with this and put it out of habit.

File: mingw-w64-jpegoptim/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Maintainer: Ray Griffin <rorgoroth-dev@outlook.com>
+
+_realname=jpegoptim
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.4.4
+pkgrel=1
+pkgdesc="Utility to optimize jpeg files"
+arch=('any')
+url="http://www.kokkonen.net/tjko/projects.html"
```
Comment (rorgoroth) [inline mingw-w64-jpegoptim/PKGBUILD]: Thanks, done.

**Pull request merged**